scan/lib/scan/test_command_generator.rb

```diff
@@ -83,7 +89,7 @@ def pipe
           formatter << "--test"
         end
 
-        ["| tee '#{xcodebuild_log_path}' | xcpretty #{formatter.join(' ')}"]
+        pipe << ["| xcpretty #{formatter.join(' ')}"]
```

Comment:
Just to be more explicit and make it more clear for non-Ruby developers, could you add a `return pipe` after this line? 👍